public_pom.xml

```diff
@@ -48,5 +48,10 @@
             <version>1.7.36</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.luben</groupId>
+            <artifactId>zstd-jni</artifactId>
+            <version>1.5.0-1</version>
```

Comment:
should we add compile here similar to the other two?

Reply:
`compile` is the default, so in order to be consistent, we should either remove it everywhere and add it everywhere.